Formula/f/flarectl.rb

```diff
@@ -1,6 +1,7 @@
 class Flarectl < Formula
   desc "CLI application for interacting with a Cloudflare account"
   homepage "https://github.com/cloudflare/cloudflare-go/tree/master/cmd/flarectl"
+  # no flarectl v2 for use yet
```

Comment:
What does "for use yet" mean?

Reply:
cloudflare-go has totally changed the structure, so there is no v2 atm

> hey! the v2 branch doesn't have flarectl. we are in the process of rewriting a CLI tool that will supersede flarectl.
> for now, this should be pinned to < 2 to keep using it.

maybe I should leave as TODO?

Reply:
It reads to me like flarectl will never get a V2 and instead there will be a new tool eventually.

Reply:
the library will have a new version of flarectl however, i'm unsure _if_ it will be called flarectl. i would say we can only assume it will exist < 2.x. i can always open a PR to add whatever the new thing ends up being when the time comes. 

v2 is powered by openapi schemas for the generation and the CLI portion is still in the works.

Reply:
indeed, I just update the note to make it a bit more clear